src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -630,17 +649,47 @@ static void uct_rc_mlx5_iface_common_dm_tl_cleanup(uct_mlx5_dm_data_t *data)
 
 #if IBV_HW_TM
 void uct_rc_mlx5_init_rx_tm_common(uct_rc_mlx5_iface_common_t *iface,
+                                   const uct_rc_iface_common_config_t *config,
                                    unsigned rndv_hdr_len)
 {
     uct_ib_md_t *md       = uct_ib_iface_md(&iface->super.super);
     unsigned tmh_hdrs_len = sizeof(struct ibv_tmh) + rndv_hdr_len;
+    ucs_status_t status;
 
     iface->tm.eager_desc.super.cb = uct_rc_mlx5_release_desc;
-    iface->tm.eager_desc.offset   = sizeof(struct ibv_tmh)
-                                    - sizeof(uct_rc_mlx5_hdr_t)
-                                    + iface->super.super.config.rx_headroom_offset;
-
     iface->tm.rndv_desc.super.cb  = uct_rc_mlx5_release_desc;
+
+    if (UCT_RC_MLX5_MP_ENABLED(iface)) {
+        iface->tm.eager_desc.offset = sizeof(struct ibv_tmh) +
+                                      iface->super.super.config.rx_headroom_offset;
+        iface->tm.am_desc.offset   += sizeof(uct_rc_mlx5_hdr_t);
+        status = uct_iface_mpool_init(&iface->super.super.super,
+                                      &iface->tm.mp.tx_mp,
+                                      sizeof(uct_rc_iface_send_desc_t) +
+                                      UCT_RC_MLX5_TAG_BCOPY_MAX,
+                                      sizeof(uct_rc_iface_send_desc_t),
+                                      UCS_SYS_CACHE_LINE_SIZE,
+                                      &config->super.tx.mp,
+                                      iface->super.config.tx_qp_len,
+                                      uct_rc_iface_send_desc_init,
+                                      "tag_eager_send_desc");
+        if (status != UCS_OK) {
+            return;
+        }
+        iface->tm.bcopy_mp  = &iface->tm.mp.tx_mp;
+        iface->tm.max_zcopy = uct_ib_iface_port_attr(&iface->super.super)->max_msg_sz;
+        iface->tm.max_bcopy = UCT_RC_MLX5_TAG_BCOPY_MAX;
+        ucs_debug("Multi-Packet WQ config: stride size %d, strides per WQE %d",
+                  iface->super.super.config.seg_size, iface->tm.mp.num_strides);
+    } else {
+        iface->tm.eager_desc.offset = sizeof(struct ibv_tmh)
+            - sizeof(uct_rc_mlx5_hdr_t)
```

Comment:
should 686-687 lines be written in the same column as a 685? i.e. should they have the same offset from the beginning of the line?